Makefile

```diff
@@ -0,0 +1,6 @@
+all: build
```

Comment:
I guess we should make sure that we have all dependencies as well if not download them.